mods/cnc/maps/gdi06/rules.yaml

```diff
@@ -163,7 +163,7 @@ RMBO.easy:
 		HP: 300
 	SelfHealing:
 		Delay: 10
-		HealIfBelow: 50%
```

Comment:
Is this the last use of the `%` syntax for loading floats?  We should drop that from FieldLoader if it is.